components/PetHappiness.js

```diff
@@ -0,0 +1,15 @@
+export default function PetHappiness({ needs }) {
+  const averageNeed =
+    (needs.hunger + needs.energy + needs.entertainment) / 3;
```

Comment:
```suggestion
const values = Object.values(needs);
const averageNeed = values.reduce((sum, val) => sum + val, 0) / values.length;
```
Hardcoding is fine… until it's not. If you ever add more needs (say, `social`, `hygiene`, or `idontknowwhat` 😄), this will automatically account for them. Might be overkill now, but future-you might thank you.

Reply:
added, great suggestion, thanks! 👍